Project.toml

```diff
@@ -4,15 +4,21 @@ authors = ["Tim DuBois", "George Datseris", "Ali Vahdati"]
 version = "4.0.5"
 
 [deps]
+AgentsPlots = "7820620d-95fb-4739-93b3-b0a14dd83f9a"
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 GraphRecipes = "bd48cda9-67a9-57be-86fa-5b3c104eda73"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 OpenStreetMapX = "86cd37e6-c0ff-550b-95fe-21d72c8d4fc9"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
What happened here? None of these should be added as Agents dependencies. Also AgentsPlots is deprecated.

Reply:
`DataStructures` is an added dependency for A* Pathfinding in #405. It appears as if the rest were added in [this commit](https://github.com/JuliaDynamics/Agents.jl/pull/412/commits/fe7a516ee3c2cbc761c35139670a3f09ad2f8120)

Reply:
Okay, the rest should be removed. If they are dependencies for building documentation then they can be put in the `docs/Project.toml` instead. Although I doubt revise and progress meters are needed to build the docs.